tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -135,7 +193,9 @@ def pre_insert_agents():
 
 @pytest.fixture(scope='function')
 def insert_agents_test():
-    """Insert agents. Only used for the agent queries"""
+    '''
+    Insert agents. Only used for the agent queries.
+    '''
```

Comment:
I would not change this line because it follows the Google style correctly, with a one-line summary. Also, try to use `"""` instead.
```suggestion
   """Insert agents. Only used for agent queries."""
```

Reply:
Done in the following [commit](https://github.com/wazuh/wazuh-qa/commit/5ceb7bd05d809dc92ccde0df5d314a5914036df8)